bim2sim/plugins/PluginEnergyPlus/bim2sim_energyplus/task/ep_create_idf.py

```diff
@@ -1453,23 +1453,35 @@ def idf_validity_check(idf):
 
         # remove erroneous fenestration surfaces which do may crash
         # EnergyPlus simulation
-        fenestration = idf.idfobjects['FENESTRATIONSURFACE:DETAILED']
-        for f in fenestration:
-            if not f.Building_Surface_Name:
-                logger.info('Removed Fenestration: %s' % f.Name)
-                idf.removeidfobject(f)
-            fbco = f.Building_Surface_Name
-            bs = idf.getobject('BUILDINGSURFACE:DETAILED', fbco)
-            if bs.Outside_Boundary_Condition == 'Adiabatic':
-                logger.info('Removed Fenestration: %s' % f.Name)
-                idf.removeidfobject(f)
-        for f in fenestration:
-            fbco = f.Building_Surface_Name
-            bs = idf.getobject('BUILDINGSURFACE:DETAILED', fbco)
-            if bs.Outside_Boundary_Condition == 'Adiabatic':
-                logger.info(
-                    'Removed Fenestration in second try: %s' % f.Name)
-                idf.removeidfobject(f)
+        fenestrations = idf.idfobjects['FENESTRATIONSURFACE:DETAILED']
+        removed_fenestrations = []
```

Comment:
removed_fenestrations = [] is obsolete, please remove this line.

Reply:
solved with 6df13dda